rust/benchmarks/benches/criterion.rs

```diff
@@ -0,0 +1,10 @@
+use benchmarks::RunArgs;
+use criterion::{criterion_group, criterion_main, Criterion};
+
```

Comment:
We have the tradition of writing the command line that executes this program as a comment in the code